dev/how-to/deploy/orchestrated/ansible.md

```diff
@@ -59,15 +59,19 @@ Log in to the Control Machine using the `root` user account, and run the corresp
 
 - If you use a Control Machine installed with CentOS 7, run the following command:
 
-    ```
-    # yum -y install epel-release git curl sshpass
-    # yum -y install python-pip
+    {{< copyable "shell-root" >}}
+
+    ```shell
+    yum -y install epel-release git curl sshpass && \
+    yum -y install python2-pip
     ```
 
 - If you use a Control Machine installed with Ubuntu, run the following command:
 
-    ```
-    # apt-get -y install git curl sshpass python-pip
+    {{< copyable "shell-root" >}}
+
+    ```shell
+    apt-get -y install git curl sshpass python-pip
```

Comment:
@liubo0127 Do we need to change `python-pip` to `python2-pip` for this line, too?

Reply:
It's `python-pip` on Ubuntu operation system.

Reply:
Got it. Then let's keep the existing version.